internal/provider/bootstrap.go

```diff
@@ -59,8 +61,8 @@ func Bootstrap(e *pluggable.Event) pluggable.EventResponse {
 
 	logLevel := "debug"
 
-	if providerConfig.Kairos != nil && providerConfig.Kairos.LogLevel != "" {
-		logLevel = providerConfig.Kairos.LogLevel
+	if providerConfig.P2P != nil && providerConfig.P2P.LogLevel != "" {
```

Comment:
For the sake of consistency, we should either use `kairosBlockisDefined/p2pBlockDefined` here too, or get rid of that variable and use `providerConfig.P2P != nil` everywhere.

Reply:
Fixed!